pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -93,6 +94,8 @@ spec:
             - "--leader-election-lease-duration=137s"
             - "--leader-election-renew-deadline=107s"
             - "--leader-election-retry-period=26s"
+            - "--feature-gates=HonorPVReclaimPolicy=true"
+            - "--prevent-volume-mode-conversion=true"
```

Comment:
this is not required for cephfs and nfs as we don't support blockmode PVC but no harm in adding it as well :)